src/NServiceBus.AwsLambda.SQS/AwsLambdaSQSEndpointConfiguration.cs

```diff
@@ -52,30 +72,30 @@ void TrySpecifyDefaultLicense()
         }
 
         /// <summary>
-        /// Amazon SQS transport
+        /// Amazon SQS transport routing settings
         /// </summary>
-        public TransportExtensions<SqsTransport> Transport { get; }
+        public RoutingSettings<SqsTransport> RoutingSettings { get; }
+
+        /// <summary>
+        /// Amazon SQS Transport
+        /// </summary>
+        public SqsTransport Transport { get; }
 
         internal EndpointConfiguration EndpointConfiguration { get; }
-        internal PipelineInvoker PipelineInvoker { get; private set; }
+
+        internal ServerlessTransport<SqsTransport> MakeServerless()
```

Comment:
this is currently not being called by the testing setups from what I can tell, so they might not be correctly configured.

Reply:
[AwsLambdaEndpoint calls it automatically during initialization ](https://github.com/Particular/NServiceBus.AwsLambda.Sqs/blob/upgrade-to-v8/src/NServiceBus.AwsLambda.SQS/AwsLambdaEndpoint.cs#L63)

I did debug the tests and confirmed that it does get called.